navigation/navbar.tsx

```diff
@@ -1,55 +1,46 @@
-import React from 'react';
-import { NavigationContainer } from '@react-navigation/native';
-import { createStackNavigator } from '@react-navigation/stack';
-import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
-import { Ionicons } from '@expo/vector-icons';
+import React from "react";
+import { createBottomTabNavigator } from "@react-navigation/bottom-tabs";
+import { Ionicons } from "@expo/vector-icons";
 
-// screens 
-import FacilityProfile from './screens/facility-profile';
-import FacilityReviews from './screens/facility-reviews';
-import Login from './screens/login';
-import Main from './screens/main';
-import ReviewForm from './screens/review-form';
-import UserProfile from './screens/user-profile';
+// types
+import { ScreensParamList } from "./type";
 
-// screen names 
-const facilityProfileName = "Facility Profile";
-const facilityReviewsName = "Facility Reviews";
-const loginName = "Login";
+// screens
+import Main from "./screens/main";
+import ReviewForm from "./screens/review-form";
+import UserProfile from "./screens/user-profile";
+
+// screen names
 const mainName = "Main";
-const reviewFormName = "Review Form";
-const userProfileName = "User Profile";
+const reviewFormName = "ReviewForm";
+const userProfileName = "UserProfile";
 
-const Tab = createBottomTabNavigator();
+// navigation
+const Tab = createBottomTabNavigator<ScreensParamList>();
 
-export default function Navigation() {
+export default function NavBar() {
   return (
-    <NavigationContainer>
-      <Tab.Navigator
-        initialRouteName={mainName}
-        screenOptions={({ route }) => ({
-          tabBarIcon: ({ focused, color, size }) => {
-            let iconName;
-            let current = route.name;
-
-            if (current === mainName) {
-              iconName = focused ? 'home' : 'home-outline';
-
-            } else if (current === reviewFormName) {
-              iconName = focused ? 'add-circle' : 'add-circle-outline';
-
-            } else if (current === userProfileName) {
-              iconName = focused ? 'person-circle' : 'person-circle-outline';
-            }
-            return <Ionicons name={iconName} size={size} color={color} />;
-          },
-        })}> 
-
-        <Tab.Screen name={mainName} component={Main} />
+    <Tab.Navigator
+      initialRouteName={mainName} // ask about implementing login as initial page display
```

Comment:
Note: login doesn't *have* to be the initial page. for instance, someone can look at the map without logging in. we only need to log in if someone wants to write data